src/instructlab/configuration.py

```diff
@@ -307,6 +308,18 @@ def get_default_config():
             mt_bench_branch=_mtbenchbranch(taxonomy_path=DEFAULT_TAXONOMY_PATH),
             mmlu_branch=_mmlubranch(sdg_path=DEFAULT_GENERATED_FILES_OUTPUT_DIR),
         ),
+        evaluate_serve=_serve(
+            model_path=DEFAULT_MODEL_PATH,
+            backend="vllm",
+            llama_cpp=_serve_llama_cpp(
+                gpu_layers=-1,
+                max_ctx_size=5120,
```

Comment:
Why 5120? Can we add a comment to give it an explanation?

Reply:
Done, thanks!